### ISSUE TITLE
Add better support of GraphQL

### ISSUE THREAD

**Issue by Sytten** [caido/caido#209]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

GraphQL is becoming more and more common, we could enhance the experience for those users.
A few ideas (please comment below, we want to know what you need!):
- Pretty view of the query and variables
- Match APQ with their definition if we know it
- A better editor for the replay that is "semantic"

**Comment by thamzhack** [comment]
It would be really cool if we can intercept caido's GraphQL playground to deal with sites that have no introspection query like  [in this z-wink's video](https://youtu.be/JLO3bd_nVks?t=873)

**Comment by Sytten** [comment]
Good idea @thamzhack! Once the graphql playground v2 gets released we will integrate it caido and we can definitely create that feature.

**Comment by Corb3nik** [comment]
This is now done via a plugin. We don't have any plans to support this in core.
